core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -717,8 +718,15 @@ public Transaction createTransaction() {
 
     @Override
     public Transaction replaceTransaction() {
-      if (viewExists(context, ident)) {
-        throw new AlreadyExistsException("View with same name already exists: %s", ident);
+      try {
+        if (viewExists(context, ident)) {
+          throw new AlreadyExistsException("View with same name already exists: %s", ident);
+        }
+      } catch (RESTException | UnsupportedOperationException e) {
+        // don't fail if the server doesn't support views, which could be due to:
+        // 1. server or backing catalog doesn't support views
+        // 2. newer client talks to an older server that doesn't support views
+        LOG.debug("Could not check whether view {} exists: {}", ident, e.getMessage(), e);
```

Comment:
Style: typically we would use "Failed to check whether view {} exists" and leave the message to the exception trace.

Reply:
updated